cli/src/main/java/com/box/l10n/mojito/cli/command/extractioncheck/ExtractionCheckNotificationSenderSlack.java

```diff
@@ -0,0 +1,54 @@
+package com.box.l10n.mojito.cli.command.extractioncheck;
+
+import com.box.l10n.mojito.cli.command.checks.CliCheckResult;
+import com.box.l10n.mojito.thirdpartynotification.ThirdPartyNotificationMessageBuilder;
+import com.box.l10n.mojito.thirdpartynotification.ThirdPartyNotificationMessageType;
+import com.box.l10n.mojito.thirdpartynotification.ThirdPartyNotificationService;
+import com.box.l10n.mojito.thirdpartynotification.ThirdPartyNotificationType;
+import com.box.l10n.mojito.thirdpartynotification.phabricator.ThirdPartyNotificationMessageBuilderPhabricator;
+import com.box.l10n.mojito.thirdpartynotification.slack.ThirdPartyNotificationMessageBuilderSlack;
+import com.google.common.base.Strings;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Configurable
+public class ExtractionCheckNotificationSenderSlack extends ExtractionCheckNotificationSender {
+
+    @Autowired
+    ThirdPartyNotificationService thirdPartyNotificationService;
```

Comment:
Wondering, what's the benefit of going through that service again?

it seems to be doing that:
```
       ---> slack                                 slack ---> 
CLI                    ---> generic --->
       ---> phab                                  phab --->
```

while it could just be?
```
       ---> slack
CLI      
       ---> phab

```